src/current/_includes/v23.2/sql/privileges.md

```diff
@@ -5,7 +5,7 @@ Privilege | Levels | Description
 `CANCELQUERY` | System | Grants the ability to cancel queries.
 `CHANGEFEED` | Table | Grants the ability to create [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}) on a table.
 `CONNECT` | Database | Grants the ability to view a database's metadata, which consists of objects in a database's `information_schema` and `pg_catalog` system catalogs. This allows the role to view the database's table, schemas, user-defined types, and list the database when running `SHOW DATABASES`. The `CONNECT` privilege is also required to run backups of the database.
-`CREATE` | Database, Schema, Table, Sequence | Grants the ability to create objects at the database, schema, table, or sequence level. When applied at the database level, grants the ability to configure [multi-region zone configs]({% link {{ page.version.version }}/zone-config-extensions.md %}).
+`CREATE` | Database, Schema, Table, Sequence | Grants the ability to create objects at the database, schema, table, or sequence level. When applied at the database level, grants the ability to configure [multi-region zone configs]({% link {{ page.version.version }}/zone-config-extensions.md %}). <br /><br /> In CockroachDB v23.2 and later, the [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.auth.public_schema_create_privilege.enabled`. This setting controls whether users receive `CREATE` privileges on the public schema or not. The setting applies at the time that the [public schema is created]({% link {{ page.version.version }}/create-schema.md %}), which happens whenever [a database is created]({% link {{ page.version.version }}/create-database.md %}). The setting is `true` by default, but can be set to `false` for increased compatibility with [Postgres version 15](https://www.postgresql.org/about/news/postgresql-15-released-2526/) as described in [this commit](https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=b073c3ccd06e4cb845e121387a43faa8c68a7b62).
```

Comment:
i don't think anywhere else needs changes. but one quick edit here:
> In CockroachDB v23.2 and later, the [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.auth.public_schema_create_privilege.enabled`. This setting controls whether users receive `CREATE` privileges on the public schema or not.

should be 
> In CockroachDB v23.2 and later, the [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.auth.public_schema_create_privilege.enabled` controls whether users receive `CREATE` privileges on the public schema or not.

Reply:
thanks @rafiss ! guess that typo snuck in via some edits :-) 

this is updated now to your suggested text